dali/base/dautils.cpp

```diff
@@ -1014,8 +1023,19 @@ void CDfsLogicalFileName::setExternal(const char *location,const char *path)
         setExternal(rfn);  // overrides ip
         return;
     }
-    StringBuffer str(EXTERNAL_SCOPE "::");
-    str.append(location);
+
+    StringBuffer str;
+    if (planeName.isEmpty())
```

Comment:
it would be better not to inro. a new member, and/or alter setExternal to cope with both.
I think setPlaneExternal should look like this:
```
void CDfsLogicalFileName::setPlaneExternal(const char *planeName,const char *path)
{
    if (isEmptyString(path))
        return;
    if (isPathSepChar(path[0])&&(path[0]==path[1]))
        throw .... // not expected..
    StringBuffer str(PLANE_SCOPE "::");
    str.append(planeName);
    // call common function that setExternal calls too
}


```
since //ip/ for the path for setPlaneExternal

The common part (that is currentlly after     str.append(location); ), can be extract to a static func. that both setExternal and setPlaneExternal calls.